.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@mattgoud please remove this file ;-) it's dependent from your IDE configuration and we don't commit this kind of files in the repository.

Reply:
ok @nicosomb done